cli-commands/init/LimeFactory.sol

```diff
@@ -0,0 +1,22 @@
+pragma solidity ^0.4.24;
+
+contract LimeFactory {
+
+    event FreshLime(string name, uint energy);
+
+    struct Lime {
+        string name;
+        uint8 carbohydrates;
+        uint8 fat;
+        uint8 protein;
+        uint8 energy;
+    }
+
+    Lime[] public limes;
+
+    function createLime(string _name, uint8 _carbohydrates, uint8 _fat, uint8 _protein) internal {
+        uint8 energy = _carbohydrates * 3 + _fat * 2 + _protein;
```

Comment:
Although I love it, we should remove this line and the energy as we would need to add the SafeMath library which would complicate things greatly